melos.yaml

```diff
@@ -23,6 +23,11 @@ command:
   bootstrap:
     hooks:
       post: melos run format
+  version:
+    hooks:
+      preCommit: |
+        melos bootstrap
+        git add packages/app/pubspec.lock
```

Comment:
why do we need this?
Even if the lock file changes we do not expect to publish it so let's keep renovate doing this.

Reply:
This is needed because we point to the local packages using the package_overrides. Otherwise the diff will appear all the time until renovate runs. This is also not breaking us while to packages are not yet published due to pointing to the local versions.